templates/config.yaml.erb

```diff
@@ -5,6 +5,21 @@
 :server: '<%= @server %>'
 :port: <%= @port %>
 
+# Should --fetch-remote-resources be added to `oscap xccdf eval` command
+:fetch_remote_resources: <%= @fetch_remote_resources %>
+
+# HTTP proxy server for downloading remote resources
+<%- if @http_proxy_server -%>
```

Comment:
I think you can omit the conditions here and just have:
```
:http_proxy_server: <%= @http_proxy_server %>
```

Same for the `:http_proxy_port` below.